tests/forms_tests/field_tests/test_jsonfield.py

```diff
@@ -0,0 +1,100 @@
+import uuid
+
+from tests.test_utils.json import CustomDecoder
+
+from django import forms
+from django.core import exceptions
+from django.core.serializers.json import DjangoJSONEncoder
+from django.test import SimpleTestCase
+
+
+class TestFormField(SimpleTestCase):
+
+    def test_valid(self):
+        field = forms.JSONField()
+        value = field.clean('{"a": "b"}')
+        self.assertEqual(value, {'a': 'b'})
+
+    def test_valid_empty(self):
+        field = forms.JSONField(required=False)
+        value = field.clean('')
+        self.assertIsNone(value)
+
+    def test_invalid(self):
+        field = forms.JSONField()
+        with self.assertRaises(exceptions.ValidationError) as cm:
```

Comment:
Why not `assertRaisesMessage`?

Reply:
Good call, thanks. I mostly copied the form tests from `contrib.postgres` and didn't notice that.